packages/pv-create-component/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "pv-create-component",
```

Comment:
"@pro-vision/pv-create-component" would be a more consistent naming

Reply:
updated